tests/console/hostname.pm

```diff
@@ -22,6 +22,8 @@ sub run() {
 
     script_run "hostname";
     assert_screen("hostname-$hostname");
+    script_run "systemctl restart wicked.service";
+    sleep 10;
```

Comment:
I have dynamic DNS, if I restart wicked, it sends a new DHCP request with new hostname, and other DNS gets updated